### ISSUE TITLE
MvxMasterDetailPage doesn't work on UWP

### ISSUE THREAD

**Issue by Antonio2103** [MvvmCross/MvvmCross#4311]
## 🐛 Bug Report

After migrated to Mvx 8.0.2 from Mvx 7.1.2, MvxMasterDetailPage doesn't work properly on UWP, in fact, master page (menu) is blank. No issue on IOS and Android 

### Expected behavior
Master Page should be visible.

### Reproduction steps
Create a new project with MvxMasterDetailPage with some element in menu page.

### Configuration
No particular configuration is needed

**Version:** 8.0.2

**Platform:** 
- [ ] :iphone: iOS
- [ ] :robot: Android
- [ ] :checkered_flag: WPF
- [X] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

**Comment by pwisk** [comment]
I have the same problem in my app in a UWP project.

**Comment by levi930** [comment]
I have same things ,maybe like this 
https://github.com/MvvmCross/MvvmCross/issues/4321

**Comment by ccanbek89** [comment]
Currently experiencing the same issue.

**Comment by gerfen** [comment]
I have the same issue.  I've cloned a demo app to demonstrate the issue.  The working code based on MvvmCross 6.2.2 is [here](https://github.com/gerfen/MvvmCrossDemo/tree/master).  The broken code based on MvvmCross 8.0.2 is [here](https://github.com/gerfen/MvvmCrossDemo/tree/xamarin-forms-5)

**Comment by gerfen** [comment]
If someone can point me in the right direction on how to address this issue, I can work on a PR.  At the moment I'm not familiar enough with MvvMCross to know where to start.  I can see the UI elements for the MasterDetailPage are rendered but they are not associated to the flyout as shown in the images below.

![image](https://user-images.githubusercontent.com/5280484/144915779-4cb8ee00-d597-4f33-a310-c348fb1554c3.png)

![image](https://user-images.githubusercontent.com/5280484/144915883-1880e748-ea8b-4c2c-a007-65511a771982.png)

**Comment by Antonio2103** [comment]
> If someone can point me in the right direction on how to address this issue, I can work on a PR. At the moment I'm not familiar enough with MvvMCross to know where to start. I can see the UI elements for the MasterDetailPage are rendered but they are not associated to the flyout as shown in the images below.
> 
> ![image](https://user-images.githubusercontent.com/5280484/144915779-4cb8ee00-d597-4f33-a310-c348fb1554c3.png)
> 
> ![image](https://user-images.githubusercontent.com/5280484/144915883-1880e748-ea8b-4c2c-a007-65511a771982.png)

Hi, the issue is not resolved and I don't know any workaround.

**Comment by Antonio2103** [comment]
MvvmCross, any idea?
Thanks

**Comment by levi930** [comment]
感谢,已经收到!

**Comment by Cheesebaron** [comment]
It is unclear from the description what you are doing. Provide a minimal repro sample and I can have a look.

**Comment by Antonio2103** [comment]
Hi @Cheesebaron 

I attached an example project. 
As you can see, in a master detail app no menu elements are visible on UWP platform. In this example I expect application show "Home" and "Contacts" opening menu tapping/clicking on hamburger menu. On IOS and Android it works, but not in UWP

So, on IOS (Android is the same):
![Simulator Screen Shot - iPhone 12 Pro - 2022-02-01 at 15 17 07](https://user-images.githubusercontent.com/79101865/151986464-0abfa488-5983-40ae-bc44-3e41f7dc7414.png)

On UWP blank menu instead:
<img width="1383" alt="Screenshot 2022-02-01 at 15 18 53" src="https://user-images.githubusercontent.com/79101865/151986685-863d3499-4682-411a-ba74-f34f4dc72c2d.png">

 
[MasterDetailExample.zip](https://github.com/MvvmCross/MvvmCross/files/7979125/MasterDetailExample.zip)

Thank you

**Comment by Antonio2103** [comment]
Sorry, wrong UWP screenshot on previous post
<img width="1144" alt="Screenshot 2022-02-01 at 15 34 01" src="https://user-images.githubusercontent.com/79101865/151988134-8aa01c56-c137-46f6-a727-f3cc2359dd16.png">

**Comment by Antonio2103** [comment]
Hi, I cannot update to mvvmcross 8.0.2 until this issue persists. There are any news about it? 
Thanks

**Comment by levi930** [comment]
感谢,已经收到!

**Comment by Antonio2103** [comment]
So, no solution for this issue?
Thanks

**Comment by Cheesebaron** [comment]
No, been focusing on other stuff. If you want this resolved quicker. Create a PR and fix it yourself.
